tcms/testruns/templates/testruns/get.html

```diff
@@ -331,6 +332,12 @@ <h2 class="card-pf-title">
                                         <span class="fa fa-search"></span>
                                     </button>
                                 </div>
+                                <div class="form-group">
+                                    <div class="list-view-pf-checkbox hidden-print" style="margin-top: 0; margin-bottom: 0">
+                                        <input type="checkbox" class="assigned-to-me-checkbox">
+                                        <span>{% trans 'Assigned to me' %}</span>
+                                    </div>
```

Comment:
I think a Yes/No switch button after the label would be more appropriate here, not sure how it will look like.